### PR TITLE
Removed domain from wp-cli host

### PR DESCRIPTION
**Pull request by Mte90**
As discussed on https://github.com/Varying-Vagrant-Vagrants/custom-site-template/pull/50 and tested seems that doesn't ask the password in this way.

### CODE REVIEW

**Comment by tomjn** [comment]
@Mte90 this change will make it `ssh` to the domain `vagrant`, there must be  a domain

**Comment by Mte90** [comment]
Screenshot_20210112_181309

I did in that way as you can see in the screen and didn't asked for the password.

File: provision/vvv-init.sh [hunk 1]
```diff
@@ -209,10 +209,10 @@ setup_cli() {
   echo "# auto-generated file" > "${VVV_PATH_TO_SITE}/wp-cli.yml"
   echo "path: \"${PUBLIC_DIR}\"" >> "${VVV_PATH_TO_SITE}/wp-cli.yml"
   echo "@vvv:" >> "${VVV_PATH_TO_SITE}/wp-cli.yml"
-  echo "  ssh: vagrant@${DOMAIN}" >> "${VVV_PATH_TO_SITE}/wp-cli.yml"
+  echo "  ssh: vagrant" >> "${VVV_PATH_TO_SITE}/wp-cli.yml"
   echo "  path: ${PUBLIC_DIR_PATH}" >> "${VVV_PATH_TO_SITE}/wp-cli.yml"
   echo "@${VVV_SITE_NAME}:" >> "${VVV_PATH_TO_SITE}/wp-cli.yml"
-  echo "  ssh: vagrant@${DOMAIN}" >> "${VVV_PATH_TO_SITE}/wp-cli.yml"
+  echo "  ssh: vagrant" >> "${VVV_PATH_TO_SITE}/wp-cli.yml"
```
Comment (tomjn) [inline provision/vvv-init.sh:215]: These changes cripple WP CLI functionality on all environments. It also does not make sense as a change, we don't add a `vagrant` domain  to the hosts  file, so how would it know where the VM is?

**Review by tomjn — CHANGES_REQUESTED** [review]
The PR depends on `vagrant` working as a host name, but we don't add that hostname, it does not work in tests

**Comment by tomjn** [comment]
@Mte90 this does not work on MacOS, and as a change does not make sense. Wherever you are getting the `vagrant` host name from, it is not VVV

**Comment by Mte90** [comment]
I agree it isn't from VVV, also from ssh file there isn't that domain configured.
Also it isn't on hosts, so maybe on linux is different?
It is vagrant itself that maybe expose it in the internal dns?

**Comment by tomjn** [comment]
If `vagrant` exposed it via internal DNS then `ssh vagrant@vagrant` would work, it would also fall apart once more than one VM was created, e.g. a vagrant file that spun up multiple machines

**Pull request merged**